### PR TITLE
Fix Issues After Remove createErpRedeemScript method from ErpFederationRedeemScriptParser in Bj-thin

### PR DESCRIPTION
**Pull request by wilmerrootstock**
## Description
After removing the `createErpRedeemScript` method from `ErpFederationRedeemScriptParser` in `Bitcoinj-thin`, please take a look at `RSKj` to see the impact this has on it.

Fix classes using the `createErpRedeemScript` method from `ErpFederationRedeemScriptParser` using the redeem script builder in `RSKj`.

## Motivation and Context
This relates to the Refactors to add `SegwitRedeemScriptParser`.

## How Has This Been Tested?
Unit Tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Review by jeremy-then — APPROVED** [review]
LGTM.

File: rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java [hunk 1]
```diff
@@ -155,4 +157,23 @@ private static void signLegacyTransactionInputFromP2shMultiSig(BtcTransaction tr
             input.setScriptSig(inputScriptSig);
         }
     }
+
+    public static List<BtcECKey> getDefaultPublicKeys() {
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java:161]: What's the use for this? Seems to be very specific

Comment (wilmerrootstock) [reply rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java:161]: This method gets the default public keys. It'll be use only for tests.

**Review by apancorb — APPROVED** [review]
LGTM

File: rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java (hunk 1 above)
Comment (wilmerrootstock) [reply rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java]: Done! Started using the `getBtcPublicKeys` method from the active federation. Reversed the changes on `BitcoinTestUtils.`

File: rskj-core/src/test/java/co/rsk/peg/PegUtilsLegacyTest.java [hunk 2]
```diff
@@ -57,6 +61,15 @@ void setupConfig() {
         bridgeConstantsMainnet = BridgeMainNetConstants.getInstance();
         federationConstantsMainnet = bridgeConstantsMainnet.getFederationConstants();
         networkParameters = bridgeConstantsRegtest.getBtcParams();
+        Federation activeFederation = FederationTestUtils.getGenesisFederation(federationConstantsMainnet);
+        int defaultThreshold = activeFederation.getBtcPublicKeys().size() / 2 + 1;
+        int emergencyThreshold = federationConstantsMainnet.getErpFedPubKeysList().size() / 2 + 1;
+        ErpRedeemScriptBuilder nonStandardErpRedeemScriptBuilder = new NonStandardErpRedeemScriptBuilder();
```
Comment (nathanieliov) [inline rskj-core/src/test/java/co/rsk/peg/PegUtilsLegacyTest.java:67]: Please use `NonStandardErpRedeemScriptBuilderFactory` instead.

**Comment by nathanieliov** [comment]
@wilmerrootstock, these tests failing were already fixed in this PR: #2730 by me

**Comment by wilmerrootstock** [comment]
Fix addressed by @nathanieliov in this [PR](https://github.com/rsksmart/rskj/pull/2730)

**Pull request closed without merging**